x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -42,8 +44,8 @@ export const policyIndexPattern = 'metrics-endpoint.policy-*';
 export const telemetryIndexPattern = 'metrics-endpoint.telemetry-*';
 
 // File storage indexes supporting endpoint Upload/download
-export const FILE_STORAGE_METADATA_INDEX = '.fleet-endpoint-files';
-export const FILE_STORAGE_DATA_INDEX = '.fleet-endpoint-file-data';
+export const FILE_STORAGE_METADATA_INDEX = getFileMetadataIndexName('endpoint');
```

Comment:
Hey @pzl - FYI: I'm not using an exposed function out of fleet to calculate the Index names based on the pattern they have defined there.